pkg/service/service.go

```diff
@@ -219,6 +224,84 @@ func ListWithDetailedStatus(client *occlient.Client, applicationName string) (Se
 	}, nil
 }
 
+// ListOperatorServices lists all operator backed services
+func ListOperatorServices(client *kclient.Client) error {
+	glog.V(4).Info("Getting list of services")
+
+	// First let's get the list of all the operators in the namespace
+	csvs, err := client.GetClusterServiceVersionList()
+	if err != nil {
+		return errors.Wrap(err, "Unable to list operator backed services")
+	}
+
+	var allCRInstances []unstructured.Unstructured
+
+	// let's get the Services a.k.a Custom Resources (CR) defined by each operator, one by one
+	for _, csv := range csvs.Items {
+		glog.V(4).Infof("Getting services started from operator: %s\n", csv.Name)
+		customResources := client.GetCustomResourcesFromCSV(csv)
+
+		// list and write active instances of each service/CR
+		instances, err := getCRInstances(client, customResources)
+		if err != nil {
+			return err
+		}
+
+		// assuming there are more than one instances of a CR
+		allCRInstances = append(allCRInstances, instances...)
+	}
+
+	displayServices(allCRInstances)
+
+	return nil
+}
+
+func getGVRFromCR(cr olm.CRDDescription) (group, version, resource string) {
+	version = cr.Version
+
+	gr := strings.SplitN(cr.Name, ".", 2)
+	resource = gr[0]
+	group = gr[1]
```

Comment:
this can panic maybe throw an error

Reply:
A CR name is generally something like `etcdclusters.etcd.database.coreos.com` or `cockroachdbs.charts.helm.k8s.io`. And we're splitting it into two parts. I'm sure this won't panic.

Reply:
But still I think it's good to be prepared for some anomalies

Reply:
Would a `len` check for `gr` be good?

Reply:
Yeah that would do . If the length is less than 2, just throw an error.

Reply:
Done.